reference/3.0/Microsoft.PowerShell.Core/About/about_CommonParameters.md

```diff
@@ -507,18 +550,10 @@ Mode                LastWriteTime     Length Name
 ----                -------------     ------ ----
```

Comment:
@zjalexander,
All those are nice to have fixes; that are outside the user request.
I had just addressed the user request and fixed the sample issues.
Shouldn't we be providing the users with the updated content they need?